src/main/java/com/stripe/model/UsageRecord.java

```diff
@@ -0,0 +1,86 @@
+package com.stripe.model;
+
+import com.stripe.exception.APIConnectionException;
+import com.stripe.exception.APIException;
+import com.stripe.exception.AuthenticationException;
+import com.stripe.exception.CardException;
+import com.stripe.exception.InvalidRequestException;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.Map;
+
+public class UsageRecord extends APIResource implements HasId {
+  String id;
+  String object;
+  Boolean livemode;
+  Long quantity;
+  String subscriptionItem;
+  Long timestamp;
+
+  @Override
+  public String getId() {
+    return id;
+  }
+
+  public void setId(String id) {
+    this.id = id;
+  }
+
+  public String getObject() {
+    return object;
+  }
+
+  public void setObject(String object) {
+    this.object = object;
+  }
+
+  public Boolean getLivemode() {
+    return livemode;
+  }
+
+  public void setLivemode(Boolean livemode) {
+    this.livemode = livemode;
+  }
+
+  public Long getQuantity() {
+    return quantity;
+  }
+
+  public void setQuantity(Long quantity) {
+    this.quantity = quantity;
+  }
+
+  public String getSubscriptionItem() {
+    return subscriptionItem;
+  }
+
+  public void setSubscriptionItem(String subscriptionItem) {
+    this.subscriptionItem = subscriptionItem;
+  }
+
+  public Long getTimestamp() {
+    return timestamp;
+  }
+
+  public void setTimestamp(Long timestamp) {
+    this.timestamp = timestamp;
+  }
+
+  /**
+   * Create a new usage record associated with a subscription item.
+   * @param subscriptionItem Id of the subscription item
+   * @param params The quantity, the timestamp and the conflict behaviour (action)
+   * @param options Request options
+   * @return The created usage record
+   */
+  public static UsageRecord create(String subscriptionItem,
```

Comment:
Hm, I think it's pretty conventional for every `create` to use a `params` object (see `ag 'public.* create\('`). Mind converting this over?

Reply:
In this case the `subscriptionItem` is required to build the URL, so I think it's nice to have this documented in the signature? If we would pass a `params` object then we would either need to check for a `subscription_item` key at runtime and throw an exception if something goes wrong, or it will result in weird errors for the user if forgotten if we default to `""` (=> either `404` or invalid url?)

Reply:
@alexander-stripe 

Yeah, I agree it'd be nice to get this check in client-side. Given that that there isn't even a single example of a `create` that doesn't take `params` elsewhere though, I don't think we should break the pattern.

Would it make more sense if the action was on subscription item? We have methods for resource-specific actions elsewhere in the library (see `refund` on `ApplicationFee` for example).

Reply:
Moving it to the subscription item is also not optimal as you'd need to load the subscription item just to report usage. 

I'll remove the `subscriptionItem` param and add a client side check.